scripts/release_ppa.sh

```diff
@@ -89,7 +104,7 @@ Priority: extra
 Maintainer: Christian (Buildserver key) <builds@ethereum.org>
 Build-Depends: debhelper (>= 9.0.0),
                cmake,
-               g++-4.8,
+               g++-6,
```

Comment:
This seems to break most builds, but I cannot run this script on my system because it does not have this build dependency...